pull request Maioli Jacopo 17/03/2025

JacopoMaioli opened the pull request.

JacopoMaioli committed:
prova

prova

JacopoMaioli committed:
add Lifestyle: Smoking in tabacco use , in sheet patient,

JacopoMaioli committed:
add lifestile smoking in smoking , in  sheet patient

JacopoMaioli committed:
Revert "add Lifestyle: Smoking in tabacco use , in sheet patient,"

This reverts commit 4c7d56505c6efd040d6b4f9bb12637a9dfd361b7.

JacopoMaioli committed:
update cell M11 sheet Diagnosis

update cell M11 sheet Diagnosis

JacopoMaioli committed:
update name value sett cell m11 sheet Diagnosis

cancer diagnosis to cancer hystology subgroup

JacopoMaioli committed:
Merge branch 'master' of https://github.com/JacopoMaioli/idea4rc

JacopoMaioli committed:
correction cell M11

ho lasciato degli spazi nell'update della cella M11 , qui ci sono le correzioni

JacopoMaioli committed:
creation cancer hystology , from sheet Diagnosis cell M9 sheet Diagnosisl

JacopoMaioli committed:
creation sites from cell m10 sheet diagnosis

JacopoMaioli committed:
update cell 12 , sheet diagnosis

rgazzarata commented on input/fsh/terminologies/cancerEvent/cancerEventType.fsh:
Elimina il codice di prova * $athena#32948 "Stable prova" alla riga 13

rgazzarata commented on input/fsh/terminologies/cancerEvent/cancerDiagnosis.fsh:
Sposterei tutto questo value set in CANCERHYSTOLOGY.FSH.
Cambia il nome del ValueSet da ValueSet: CancerDiagnosisVS a ValueSet: CancerHystologySubgroupVS
Cambia l'id Id: cancer-diagnosis-athena-vs -> Id: cancer-histology-subgroup-athena-vs
Cambia titolo togliendo lo spazio da Title: " Cancer hystology subgroup: Athena" a Title: "Cancer hystology subgroup: Athena"
Cambia description "Codes identifing the histology subgropu of the cancer."

rgazzarata commented on input/fsh/terminologies/cancerEvent/cancerHystology.fsh:
Toglio lo spazio dal title quindi da Title:" Cancer hystology: Athena" a Title: "Cancer hystology: Athena"
Cambia la description "Codes identifying the histology of the cancer."

rgazzarata replied:
Inserire direttamente i codici N11 commentando tutti gli altri vs.

rgazzarata replied:
Il vs CancerHystolgyVs va bene.
Il vs CancerHystologySubgroupVS così sarebbe vuoto, devi mettere alla riga 28 tutti i codici che ora hai dalla riga 80 in poi (che ora il programma vedrebbe sotto CancerHystolgyVs visto che il resto è tutto commentato).
Ricontrolla comunque i codici dalla riga 80 in poi perchè ho visto degli errori (es. riga 83 codice  4247921). Forse fai prima a prendere i codici della cella N11 e inserirli direttamente alla riga 28 e commentare tutto quello che è dalla riga 70 in poi.

rgazzarata commented on input/fsh/terminologies/cancerEvent/histologyAdenocarcinoma.fsh:
il codice 4097305 va cambiato con "Adenocarcinoma, intestinal type"

rgazzarata replied:
Aggiungere lo spazio nella descrizione

rgazzarata commented on input/fsh/terminologies/cancerEvent/histologyOdontogenicCarcinoma.fsh:
Cambiare la descrizione del codice 4005818 a "Malignant odontogenic ghost cell tumor"

rgazzarata commented on input/fsh/terminologies/cancerEvent/histologySquamous.fsh:
Ricxontrolla meglio i nomi, per esempio 4078953 mi sembra che sia Squamous cell carcinoma, keratinizing

rgazzarata commented on input/fsh/terminologies/cancerEvent/histologySubgroupFmtTumours.fsh:
Controlla meglio per esempio la descrizione di 4264204 è Aggressive fibromatosis
4030130 e 4028562 non mi sembra ci siano più

rgazzarata commented on input/fsh/terminologies/cancerEvent/miscellaniousMesenchimalTumors.fsh:
Il codice 4135215 non va qui

rgazzarata commented on input/fsh/terminologies/cancerEvent/smoothMuscleTumours.fsh:
Controlla che 4014760 e 4028856 non siano in altri valueset
Leiomyosarcoma c'era già con il codice 4201415 2 righe sotto quindi devi togliere 4201415

rgazzarata commented on input/fsh/terminologies/cancerEvent/tumoursOfUncertainDifferentiation.fsh:
4328092 va tolto
4307938 va tolto
Ricontrollare meglio, 4304780 manca, 45766495 manca, 4029680 manca
Angiomyxoma - 4239956 non c'è (forse è da qualche altra parte)

rgazzarata commented on input/fsh/terminologies/cancerEvent/undifSmallRCelSarcBonAndSofTis.fsh:
Cambiare il nome a  45766528 -> Ewing sarcoma / peripheral neuroectodermal tumor

rgazzarata commented on input/fsh/terminologies/cancerEvent/vascularTumours.fsh:
Cambiare il nome a 4030138 ->  Papillary intralymphatic angioendothelioma

rgazzarata commented on input/fsh/terminologies/subjectModel/tobacco-use.fsh:
Rimuovere 1585856

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"   
```
Cambiare nome Structure of posterior wall of nasopharynx

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"   
 * $athena#4323979   "Lateral wall of nasopharynx"   
```
Cambiare nome Structure of lateral wall of nasopharynx

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"   
 * $athena#4323979   "Lateral wall of nasopharynx"   
 * $athena#4220236   "Anterior wall of nasopharynx"
```
Cambiare nome Structure of anterior wall of nasopharyn

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"   
```
Cambiare nome Structure of hypopharyngeal aspect of aryepiglottic fold

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"   
 * $athena#4102600    "Posterior wall of hypopharynx"   
```
Structure of posterior wall of hypopharynx

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"   
 * $athena#4102600    "Posterior wall of hypopharynx"   
 * $athena#4268741    "Pyriform sinus"
```
Structure of piriform recess

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"   
```
Structure of lateral wall of oropharynx

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"   
 * $athena#4045435   "Posterior wall oropharynx"   
```
Structure of posterior wall of oropharynx

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"   
 * $athena#4045435   "Posterior wall oropharynx"   
 * $athena#4235649    "Branchial cleft (site of neoplosm)"   
```
Embryonic branchial cleft

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"   
```
Glottis structure

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^experimental = false
 * $athena#4047227    "Glottis"   
-* $athena#4177005   "Supraglottis"   
+* $athena#4001049   "Supraglottis"   
```
Supraglottis structure

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * $athena#4047227    "Glottis"   
-* $athena#4177005   "Supraglottis"   
+* $athena#4001049   "Supraglottis"   
 * $athena#4002724    "Subglottis"   
```
Subglottis structure

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
-* $athena#4177005   "Supraglottis"   
+* $athena#4001049   "Supraglottis"   
 * $athena#4002724    "Subglottis"   
 * $athena#4279711    "Laryngeal cartilage"
```
Laryngeal cartilage structure

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"   
```
Eliminare

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"   
 * $athena#4077986   "Border of tongue"   
```
Structure of border of tongue

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"   
 * $athena#4077986   "Border of tongue"   
 * $athena#4172794    "Ventral surface of tongue"   
```
Structure of inferior surface of tongue

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
-* $athena#4210919    "Lower gum"   
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"   
 * $athena#4181419    "Anterior floor of mouth"
```
Structure of anterior floor of mouth

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"   
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"   
```
Structure of lateral floor of mouth

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"   
 * $athena#44498949   "Overlapping lesion of floor of mouth"   
 * $athena#4234542    "Hard palate"   
```
Hard palate structure

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#44498949   "Overlapping lesion of floor of mouth"   
 * $athena#4234542    "Hard palate"   
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"   
```
Structure of vestibule of mouth

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#4234542    "Hard palate"   
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"   
 * $athena#36769865    "Retromolar area"
```
Eliminare

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
-* $athena#44498052 "Overlapping lesion of other and unspecified parts of mouth"
+* $athena#4312102 "Structure of retromolar area of mouth"
+* $athena#2000100024 "Anterior 2/3 of tongue NOS"
+* $athena#44498031 "Overlapping lesion of other and unspecified parts of mouth"
```
Overlapping lesion of tongue

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"   
```
Structure of vermilion border of lower lip

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"   
 * $athena#4134589   "External upper lip"   
```
Structure of vermilion border of upper lip

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"   
 * $athena#4134589   "External upper lip"   
 * $athena#4027545    "External lip"   
```
Structure of vermilion border of lip

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027545    "External lip"   
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"   
+* $athena#4086480    "Mucosa of upper lip"
```
Structure of mucous membrane of upper lip

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"   
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"   
```
Structure of mucous membrane of lip

rgazzarata commented on input/fsh/terminologies/subsites.fsh:
```diff
@@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
-* $athena#36768548    "Mucosa of lower lip"   
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"   
 * $athena#4245168    "Mucosa of lip"
```
Eliminare

JacopoMaioli commented:
Grazie Roberta ,
Inizio subito
Quando scrivi eliminare ti riferisci all'ultimo codice o a tutti quelli presenti nella e-mail , esempio , in https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999178531 elimino solo * $athena#4245168 "Mucosa of lip" o tutti i codici?
Maioli Jacopo

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
From: rgazzarata ***@***.***>
Sent: Monday, March 17, 2025 5:35:15 PM
To: rgazzarata/idea4rc ***@***.***>
Cc: Jacopo Maioli - ***@***.*** ***@***.***>; Author ***@***.***>
Subject: Re: [rgazzarata/idea4rc] pull request Maioli Jacopo 17/03/2025 (PR #1)


@rgazzarata requested changes on this pull request.

________________________________

On input/fsh/terminologies/cancerEvent/cancerEventType.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998540844>:

Elimina il codice di prova * $athena#32948 "Stable prova" alla riga 13

________________________________

On input/fsh/terminologies/cancerEvent/cancerDiagnosis.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998923142>:

Sposterei tutto questo value set in CANCERHYSTOLOGY.FSH.
Cambia il nome del ValueSet da ValueSet: CancerDiagnosisVS a ValueSet: CancerHystologySubgroupVS
Cambia l'id Id: cancer-diagnosis-athena-vs -> Id: cancer-histology-subgroup-athena-vs
Cambia titolo togliendo lo spazio da Title: " Cancer hystology subgroup: Athena" a Title: "Cancer hystology subgroup: Athena"
Cambia description "Codes identifing the histology subgropu of the cancer."

________________________________

On input/fsh/terminologies/cancerEvent/cancerHystology.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998943844>:

Toglio lo spazio dal title quindi da Title:" Cancer hystology: Athena" a Title: "Cancer hystology: Athena"
Cambia la description "Codes identifying the histology of the cancer."

________________________________

On input/fsh/terminologies/cancerEvent/histologyAdenocarcinoma.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998975267>:

il codice 4097305 va cambiato con "Adenocarcinoma, intestinal type"

________________________________

On input/fsh/terminologies/cancerEvent/histologyOdontogenicCarcinoma.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998985611>:

Cambiare la descrizione del codice 4005818 a "Malignant odontogenic ghost cell tumor"

________________________________

On input/fsh/terminologies/cancerEvent/histologySquamous.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999001548>:

Ricxontrolla meglio i nomi, per esempio 4078953 mi sembra che sia Squamous cell carcinoma, keratinizing

________________________________

On input/fsh/terminologies/cancerEvent/histologySubgroupFmtTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999037688>:

Controlla meglio per esempio la descrizione di 4264204 è Aggressive fibromatosis
4030130 e 4028562 non mi sembra ci siano più

________________________________

On input/fsh/terminologies/cancerEvent/miscellaniousMesenchimalTumors.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999039429>:

Il codice 4135215 non va qui

________________________________

On input/fsh/terminologies/cancerEvent/smoothMuscleTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999058338>:

Controlla che 4014760 e 4028856 non siano in altri valueset
Leiomyosarcoma c'era già con il codice 4201415 2 righe sotto quindi devi togliere 4201415

________________________________

On input/fsh/terminologies/cancerEvent/tumoursOfUncertainDifferentiation.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999081237>:

4328092 va tolto
4307938 va tolto
Ricontrollare meglio, 4304780 manca, 45766495 manca, 4029680 manca
Angiomyxoma - 4239956 non c'è (forse è da qualche altra parte)

________________________________

On input/fsh/terminologies/cancerEvent/undifSmallRCelSarcBonAndSofTis.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999084803>:

Cambiare il nome a 45766528 -> Ewing sarcoma / peripheral neuroectodermal tumor

________________________________

On input/fsh/terminologies/cancerEvent/vascularTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999089779>:

Cambiare il nome a 4030138 -> Papillary intralymphatic angioendothelioma

________________________________

On input/fsh/terminologies/subjectModel/tobacco-use.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999098561>:

Rimuovere 1585856

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999127260>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"


Cambiare nome Structure of posterior wall of nasopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999128015>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"
 * $athena#4323979   "Lateral wall of nasopharynx"


Cambiare nome Structure of lateral wall of nasopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999128728>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"
 * $athena#4323979   "Lateral wall of nasopharynx"
 * $athena#4220236   "Anterior wall of nasopharynx"


Cambiare nome Structure of anterior wall of nasopharyn

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999130485>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"


Cambiare nome Structure of hypopharyngeal aspect of aryepiglottic fold

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999131139>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"
 * $athena#4102600    "Posterior wall of hypopharynx"


Structure of posterior wall of hypopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999131634>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"
 * $athena#4102600    "Posterior wall of hypopharynx"
 * $athena#4268741    "Pyriform sinus"


Structure of piriform recess

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999133114>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"


Structure of lateral wall of oropharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999133539>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"
 * $athena#4045435   "Posterior wall oropharynx"


Structure of posterior wall of oropharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999134172>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"
 * $athena#4045435   "Posterior wall oropharynx"
 * $athena#4235649    "Branchial cleft (site of neoplosm)"


Embryonic branchial cleft

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999135062>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"


Glottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999135740>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"


Supraglottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999136310>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"
 * $athena#4002724    "Subglottis"


Subglottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999136766>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"
 * $athena#4002724    "Subglottis"
 * $athena#4279711    "Laryngeal cartilage"


Laryngeal cartilage structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999141163>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"


Eliminare

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999144277>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"


Structure of border of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999145156>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"


Structure of inferior surface of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999150119>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"


Structure of anterior floor of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999150742>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"


Structure of lateral floor of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999151840>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"


Hard palate structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999153082>:

>  * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"


Structure of vestibule of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999154022>:

>  * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"
 * $athena#36769865    "Retromolar area"


Eliminare

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999164477>:

>  * $athena#4144069    "Vestibule of mouth"
 * $athena#36769865    "Retromolar area"
-
-* $athena#44498029 "Anterior 2/3 of tongue NOS"
-* $athena#44498052 "Overlapping lesion of other and unspecified parts of mouth"
+* $athena#4312102 "Structure of retromolar area of mouth"
+* $athena#2000100024 "Anterior 2/3 of tongue NOS"
+* $athena#44498031 "Overlapping lesion of other and unspecified parts of mouth"


Overlapping lesion of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999171640>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"


Structure of vermilion border of lower lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999172194>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"


Structure of vermilion border of upper lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999172636>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"


Structure of vermilion border of lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999176494>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"


Structure of mucous membrane of upper lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999177015>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"


Structure of mucous membrane of lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999178531>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"
 * $athena#4245168    "Mucosa of lip"


Eliminare

—
Reply to this email directly, view it on GitHub<https://github.com/rgazzarata/idea4rc/pull/1#pullrequestreview-2690126784>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BNTZVKA7MJRHDRWMEBM2ZTL2U32UHAVCNFSM6AAAAABZE3V2KGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMOJQGEZDMNZYGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

JacopoMaioli commented:
Nei commenti direttamente nel file mi riferisco al solo codice della riga specifica. Se provi a cercare quei codici in tutto l'excel non li trovi infatti. Spesso hanno sostituito codici con altri. Ti conviene quindi controllare che sia che tutti i codici che ci sono nell'excel siano nei valueset sia che nel valdese non ci siano codici in più.

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
From: Jacopo Maioli - ***@***.*** ***@***.***>
Sent: Monday, March 17, 2025 5:47:05 PM
To: rgazzarata/idea4rc ***@***.***>; rgazzarata/idea4rc ***@***.***>; Roberta Gazzarata ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [rgazzarata/idea4rc] pull request Maioli Jacopo 17/03/2025 (PR #1)

Grazie Roberta ,
Inizio subito
Quando scrivi eliminare ti riferisci all'ultimo codice o a tutti quelli presenti nella e-mail , esempio , in https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999178531 elimino solo * $athena#4245168 "Mucosa of lip" o tutti i codici?
Maioli Jacopo

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
From: rgazzarata ***@***.***>
Sent: Monday, March 17, 2025 5:35:15 PM
To: rgazzarata/idea4rc ***@***.***>
Cc: Jacopo Maioli - ***@***.*** ***@***.***>; Author ***@***.***>
Subject: Re: [rgazzarata/idea4rc] pull request Maioli Jacopo 17/03/2025 (PR #1)


@rgazzarata requested changes on this pull request.

________________________________

On input/fsh/terminologies/cancerEvent/cancerEventType.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998540844>:

Elimina il codice di prova * $athena#32948 "Stable prova" alla riga 13

________________________________

On input/fsh/terminologies/cancerEvent/cancerDiagnosis.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998923142>:

Sposterei tutto questo value set in CANCERHYSTOLOGY.FSH.
Cambia il nome del ValueSet da ValueSet: CancerDiagnosisVS a ValueSet: CancerHystologySubgroupVS
Cambia l'id Id: cancer-diagnosis-athena-vs -> Id: cancer-histology-subgroup-athena-vs
Cambia titolo togliendo lo spazio da Title: " Cancer hystology subgroup: Athena" a Title: "Cancer hystology subgroup: Athena"
Cambia description "Codes identifing the histology subgropu of the cancer."

________________________________

On input/fsh/terminologies/cancerEvent/cancerHystology.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998943844>:

Toglio lo spazio dal title quindi da Title:" Cancer hystology: Athena" a Title: "Cancer hystology: Athena"
Cambia la description "Codes identifying the histology of the cancer."

________________________________

On input/fsh/terminologies/cancerEvent/histologyAdenocarcinoma.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998975267>:

il codice 4097305 va cambiato con "Adenocarcinoma, intestinal type"

________________________________

On input/fsh/terminologies/cancerEvent/histologyOdontogenicCarcinoma.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1998985611>:

Cambiare la descrizione del codice 4005818 a "Malignant odontogenic ghost cell tumor"

________________________________

On input/fsh/terminologies/cancerEvent/histologySquamous.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999001548>:

Ricxontrolla meglio i nomi, per esempio 4078953 mi sembra che sia Squamous cell carcinoma, keratinizing

________________________________

On input/fsh/terminologies/cancerEvent/histologySubgroupFmtTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999037688>:

Controlla meglio per esempio la descrizione di 4264204 è Aggressive fibromatosis
4030130 e 4028562 non mi sembra ci siano più

________________________________

On input/fsh/terminologies/cancerEvent/miscellaniousMesenchimalTumors.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999039429>:

Il codice 4135215 non va qui

________________________________

On input/fsh/terminologies/cancerEvent/smoothMuscleTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999058338>:

Controlla che 4014760 e 4028856 non siano in altri valueset
Leiomyosarcoma c'era già con il codice 4201415 2 righe sotto quindi devi togliere 4201415

________________________________

On input/fsh/terminologies/cancerEvent/tumoursOfUncertainDifferentiation.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999081237>:

4328092 va tolto
4307938 va tolto
Ricontrollare meglio, 4304780 manca, 45766495 manca, 4029680 manca
Angiomyxoma - 4239956 non c'è (forse è da qualche altra parte)

________________________________

On input/fsh/terminologies/cancerEvent/undifSmallRCelSarcBonAndSofTis.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999084803>:

Cambiare il nome a 45766528 -> Ewing sarcoma / peripheral neuroectodermal tumor

________________________________

On input/fsh/terminologies/cancerEvent/vascularTumours.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999089779>:

Cambiare il nome a 4030138 -> Papillary intralymphatic angioendothelioma

________________________________

On input/fsh/terminologies/subjectModel/tobacco-use.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999098561>:

Rimuovere 1585856

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999127260>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"


Cambiare nome Structure of posterior wall of nasopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999128015>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"
 * $athena#4323979   "Lateral wall of nasopharynx"


Cambiare nome Structure of lateral wall of nasopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999128728>:

> @@ -209,6 +218,7 @@ Description: "Specifies the subsite for cancers occured in nasopharynx: Athena"
 * $athena#4146741   "Posterior wall of nasopharynx"
 * $athena#4323979   "Lateral wall of nasopharynx"
 * $athena#4220236   "Anterior wall of nasopharynx"


Cambiare nome Structure of anterior wall of nasopharyn

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999130485>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"


Cambiare nome Structure of hypopharyngeal aspect of aryepiglottic fold

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999131139>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"
 * $athena#4102600    "Posterior wall of hypopharynx"


Structure of posterior wall of hypopharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999131634>:

> @@ -238,6 +248,7 @@ Description: "Specifies the subsite for cancers occured in hypopharynx: Athena"
 * $athena#4262980   "Hypopharyngeal aspect of aryepiglottic fold"
 * $athena#4102600    "Posterior wall of hypopharynx"
 * $athena#4268741    "Pyriform sinus"


Structure of piriform recess

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999133114>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"


Structure of lateral wall of oropharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999133539>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"
 * $athena#4045435   "Posterior wall oropharynx"


Structure of posterior wall of oropharynx

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999134172>:

> @@ -274,6 +285,8 @@ Description: "Specifies the subsite for cancers occured in Oropharynx: Athena"
 * $athena#4078066    "Lateral wall oropharynx"
 * $athena#4045435   "Posterior wall oropharynx"
 * $athena#4235649    "Branchial cleft (site of neoplosm)"


Embryonic branchial cleft

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999135062>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"


Glottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999135740>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"


Supraglottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999136310>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"
 * $athena#4002724    "Subglottis"


Subglottis structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999136766>:

> @@ -309,9 +322,10 @@ Description: "Specifies the subsite for cancers occured in Larynx: Athena"
 * ^status = #active
 * ^experimental = false
 * $athena#4047227    "Glottis"
-* $athena#4177005   "Supraglottis"
+* $athena#4001049   "Supraglottis"
 * $athena#4002724    "Subglottis"
 * $athena#4279711    "Laryngeal cartilage"


Laryngeal cartilage structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999141163>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"


Eliminare

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999144277>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"


Structure of border of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999145156>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"


Structure of inferior surface of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999150119>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"


Structure of anterior floor of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999150742>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"


Structure of lateral floor of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999151840>:

> @@ -339,18 +353,19 @@ Description: "Specifies the subsite for cancers occured in oral cavity: Athena"
 * $athena#36769609    "Dorsal surface tongue, NOS"
 * $athena#4077986   "Border of tongue"
 * $athena#4172794    "Ventral surface of tongue"
-* $athena#4047029    "Upper gum"
-* $athena#4210919    "Lower gum"
+* $athena#4047029    "Structure of gum of maxilla"
+* $athena#4210919    "Structure of gum of mandible"
 * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"


Hard palate structure

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999153082>:

>  * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"


Structure of vestibule of mouth

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999154022>:

>  * $athena#4181419    "Anterior floor of mouth"
 * $athena#4103066    "Lateral floor of mouth"
 * $athena#44498949   "Overlapping lesion of floor of mouth"
 * $athena#4234542    "Hard palate"
-* $athena#4057160    "Cheek mucosa"
 * $athena#4144069    "Vestibule of mouth"
 * $athena#36769865    "Retromolar area"


Eliminare

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999164477>:

>  * $athena#4144069    "Vestibule of mouth"
 * $athena#36769865    "Retromolar area"
-
-* $athena#44498029 "Anterior 2/3 of tongue NOS"
-* $athena#44498052 "Overlapping lesion of other and unspecified parts of mouth"
+* $athena#4312102 "Structure of retromolar area of mouth"
+* $athena#2000100024 "Anterior 2/3 of tongue NOS"
+* $athena#44498031 "Overlapping lesion of other and unspecified parts of mouth"


Overlapping lesion of tongue

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999171640>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"


Structure of vermilion border of lower lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999172194>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"


Structure of vermilion border of upper lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999172636>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"


Structure of vermilion border of lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999176494>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"


Structure of mucous membrane of upper lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999177015>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"


Structure of mucous membrane of lip

________________________________

In input/fsh/terminologies/subsites.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r1999178531>:

> @@ -382,8 +397,8 @@ Description: "Specifies the subsite for cancers occured in lip: Athena"
 * $athena#4027546    "External lower lip"
 * $athena#4134589   "External upper lip"
 * $athena#4027545    "External lip"
-* $athena#36769700    "Mucosa of upper lip"
-* $athena#36768548    "Mucosa of lower lip"
+* $athena#4086480    "Mucosa of upper lip"
+* $athena#4245168    "Mucosa of lower lip"
 * $athena#4245168    "Mucosa of lip"


Eliminare

—
Reply to this email directly, view it on GitHub<https://github.com/rgazzarata/idea4rc/pull/1#pullrequestreview-2690126784>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BNTZVKA7MJRHDRWMEBM2ZTL2U32UHAVCNFSM6AAAAABZE3V2KGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMOJQGEZDMNZYGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

JacopoMaioli committed:
correzione errori

JacopoMaioli committed:
create Cancer Diagnosis

create cancer diagnosis from sheet HNC condition list . sarcome condition list

JacopoMaioli committed:
update description cancer diagnosis

JacopoMaioli committed:
raggruppamento con commento di cancer hystology sub group

JacopoMaioli committed:
creation CodedtestResult.fsh

cretion Vs vodedtestedresult.fsh
formed by  MitoticIndexVS
add mitotic index in vs lab test
and some correction in cancer hystology

JacopoMaioli committed:
update subsites.fsh

update subsite.fsh , i rewrite all codes from cellM12 sheet Diagnosis and comment the other codes

JacopoMaioli committed:
unpdate cancerhystologysubgroup

JacopoMaioli commented:
Buongiorno Roberta,
 ho fatto l'upload della versione corretta
Jacopo

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
From: rgazzarata ***@***.***>
Sent: Thursday, March 20, 2025 9:12:46 AM
To: rgazzarata/idea4rc ***@***.***>
Cc: Jacopo Maioli - ***@***.*** ***@***.***>; Author ***@***.***>
Subject: Re: [rgazzarata/idea4rc] pull request Maioli Jacopo 17/03/2025 (PR #1)


@rgazzarata requested changes on this pull request.

________________________________

On input/fsh/terminologies/cancerEvent/cancerHystology.fsh<https://github.com/rgazzarata/idea4rc/pull/1#discussion_r2004993936>:

Il vs CancerHystolgyVs va bene.
Il vs CancerHystologySubgroupVS così sarebbe vuoto, devi mettere alla riga 28 tutti i codici che ora hai dalla riga 80 in poi (che ora il programma vedrebbe sotto CancerHystolgyVs visto che il resto è tutto commentato).
Ricontrolla comunque i codici dalla riga 80 in poi perchè ho visto degli errori (es. riga 83 codice 4247921). Forse fai prima a prendere i codici della cella N11 e inserirli direttamente alla riga 28 e commentare tutto quello che è dalla riga 70 in poi.

—
Reply to this email directly, view it on GitHub<https://github.com/rgazzarata/idea4rc/pull/1#pullrequestreview-2701579086>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BNTZVKA7NJQX5J4GQM2TO6L2VJZ75AVCNFSM6AAAAABZE3V2KGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDOMBRGU3TSMBYGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
